components/automate-chef-io/content/docs/on-prem-builder.md

```diff
@@ -46,6 +50,35 @@ Roughly 80 GB of the disk space is designated for Chef Automate; the rest is use
 Chef Habitat Builder and the artifacts it stores. The current implementation uses Minio for
 Chef Habitat artifact storage; we do not support using Artifactory for artifact storage.
 
+### Hardware Sizing Considerations
+
+#### Environment Variables
+The frequency at which Habitat Supervisors poll the on-premises Chef Builder API for changes can be controlled by three Environment variables in the Chef Habitat Supervisor's runtime.
```

Comment:
Three environment variables control the frequency at which Habitat Supervisors poll the on-premises Chef Builder API for changes: